lineup.js

```diff
@@ -1,9 +1,16 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((total, lineup) => {
+    return total + lineup.salary
+  }, 0)
 }
 
 function getPositionCounts(lineup) {
-  return {}
+  return lineup.reduce((position, player) => {
```

Comment:
Minor naming note, `position` single makes it sound like a single thing. What you are building is a collection of all the positions that appear in the lineup and the count of how many times they appear. Names like `positions` or `positionCounts` would be better.